src/transformers/modeling_utils.py

```diff
@@ -2962,6 +3012,11 @@ def from_pretrained(
                 kwargs["skip_keys"] = model._skip_keys_device_placement
             dispatch_model(model, **kwargs)
 
+        if quantization_method_from_args == QuantizationMethod.GPTQ:
```

Comment:
My question here is how this will handle loading already quantized weights from the Hub or locally ? It seems it is quantizing the model no matter what? Or maybe there is something that is done under the hood by `quantize_model` ?

Reply:
Perfect this sounds great and it is very clear for me! Thanks for explaining